backoffice/locale/fr/LC_MESSAGES/django.po

```diff
@@ -18,95 +18,226 @@ msgstr ""
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:76
+msgid "Enrolled students"
+msgstr "Etudiants inscrits"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:44
+#, fuzzy
+msgid "Enrollment end"
+msgstr "Dates d'inscription"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:43
+#, fuzzy
+msgid "Enrollment start"
+msgstr "Dates d'inscription"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:45
+#, fuzzy
+msgid "Enrollments"
 msgstr "Dates d'inscription"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:53
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:66
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:137
+msgid "Full name:"
+msgstr "Nom complet:"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:118
+msgid "Generate"
+msgstr "Générer"
+
+msgid "Generate certificate"
+msgstr "Génération d'attestation"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:22
+msgid "Generate a certificate"
+msgstr "Générer une attestation"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:22
+msgid "Generate certificates"
+msgstr "Générer les attestations"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:21
 msgid "Generate test certificate"
-msgstr "Générer un certificat de test"
+msgstr "Générer une attestation de test"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:25
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:38
 msgid "Ident"
 msgstr "Identifiant"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:51
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:47
+
+msgid "Informations"
+msgstr "Informations sur le cours"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:67
 msgid "No result for this search pattern"
 msgstr "Pas de résultat pour cette recherche"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:85
+#: /edx/app/edxapp/fun-apps/backoffice/models.py:24
+msgid "Nom complet"
+msgstr ""
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:30
+msgid "Pending Certificate Tasks"
+msgstr "Génération d'attestations en cours"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:173
 msgid "Please confirm you really want to delete this course:"
 msgstr "Veuillez confirmer la suppression de ce cours :"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:24
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:37
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:113
+
+msgid "Please confirm do you really want to generate a certificate for course:"
```

Comment:
msgid "Please confirm that you really want to generate a certificate for course:"